examples/react-es6/readme.md

```diff
@@ -0,0 +1,35 @@
+# React TodoMVC Example
```

Comment:
Please update the readme file to contain data about your example, for example, add links and resources to webpack and ES6 also edit the title to show that this example contains a bundled version of react